pages/hydra/developer_resources/workflow_and_mediated_deposit/hyrax_1.0/workflow_cheat_sheet.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Workflow Cheat Sheet"
+keywords: Mediated Deposit, Workflow, Sipity
+categories: Workflow
+permalink: workflow_cheat_sheet.html
+folder: hydra/developer_resources/workflow_and_mediated_deposit/hyrax_1.0/workflow_cheat_sheet.md
+sidebar: home_sidebar
+tags: [development_resources]
+---
+
+# Handy commands
+
+### Get the active workflow for a given admin set
+```ruby
+a = AdminSet.first
+workflow = a.permission_template.available_workflows.where(active: true).first
+ => #<AdminSet id: "g732d8963", title: ["Laney Graduate School"], description: [], creator: ["redacted@fake.edu"], access_control_id: "ea6d2579-2f17-4e05-a348-f36716b783e5", representative_id: nil, thumbnail_id: nil>
+```
+
+### Get the workflow roles in a given workflow
+```ruby
+workflow_roles = Sipity::WorkflowRole.where(workflow_id: workflow.id)
```

Comment:
If you've already got a `Sipity::Workflow` object, I believe you can get at this using the `has_many` relationship between the two classes via `workflow_roles = workflow.workflow_roles`